AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -789,7 +789,7 @@ exakta.se,exaktacreative.se,exaktasoftware.se#$#.cookiePopup { display: none !im
 exakta.se,exaktacreative.se,exaktasoftware.se#$#.cookiePopupVisible [canvas]:not(.cookiePopup) { filter: none !important; }
 exakta.se,exaktacreative.se,exaktasoftware.se#$#html.cookiePopupVisible { overflow: auto !important; }
 lww.com##.cookie-jar
-steamgriddb.com##.noty_layout
+steamgriddb.com##.noty_none.noty_close_with_button.noty_has_progressbar.noty_theme__sgdb.noty_type__info.noty_bar
```

Comment:
Fixed and anserewed https://github.com/AdguardTeam/AdguardFilters/issues/122234#issuecomment-1160113634.